packages/react-native-sdk/src/components/Connect.tsx

```diff
@@ -0,0 +1,217 @@
+import {useFlowCurrentUser} from "@onflow/react-core"
+import {UseCrossVmTokenBalanceData} from "@onflow/react-core"
+import React, {useCallback, useMemo, useState} from "react"
+import {
+  Modal,
+  Pressable,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+} from "react-native"
+import {Profile} from "./Profile"
+
+type BalanceType = keyof UseCrossVmTokenBalanceData
+
+export type TokenConfig = {
+  symbol: string
+  name: string
+} & (
+  | {vaultIdentifier: string; erc20Address?: never}
+  | {vaultIdentifier?: never; erc20Address: string}
+)
+
+export interface ConnectProps {
```

Comment:
Are these the same as the connect component? Should it be shared? How do we make sure these don't get out of sync. Or maybe it doesn't matter if they do since they are separate components?